desktop/core/src/desktop/js/utils/hooks/useLoadData.ts

```diff
@@ -61,8 +66,16 @@ const useLoadData = <T, U = unknown>(url?: string, options?: IOptions<T, U>): IU
         const fetchUrl = localOptions?.urlPrefix ? `${localOptions.urlPrefix}${url}` : url;
         const response = await get<T, U>(fetchUrl, localOptions?.params, fetchOptions);
         setData(response);
+        setError(undefined);
+        if (localOptions?.onSuccess) {
+          localOptions.onSuccess(response);
+        }
       } catch (error) {
         setError(error instanceof Error ? error : new Error(error));
+        setData(undefined);
```

Comment:
old data will be removed do we want that? 
lets say there is table already loaded and we are calling useLoadData hook in a interval. in case of api fails, we can throw error but should we remove the tables data?

Reply:
Ah, this makes sense.